tests/test-sources/plugins/languages/treesitter/treesitter.nix

```
@@ -1,14 +1,66 @@
+{ pkgs, ... }:
 {
+  default = {
+    plugins.treesitter = {
+      enable = true;
+
+      settings = {
+        auto_install = false;
+        ensure_installed = [ ];
+        ignore_install = [ ];
+        parser_install_dir = null;
+        sync_install = false;
+
+        highlight = {
+          additional_vim_regex_highlighting = false;
+          enable = false;
+          custom_captures = { };
+          disable = null;
+        };
+
+        incremental_selection = {
+          enable = false;
+          keymaps = {
+            init_selection = "gnn";
+            node_incremental = "grn";
+            scope_incremental = "grc";
+            node_decremental = "grm";
+          };
+        };
+
+        indent = {
+          enable = false;
+        };
+      };
+    };
+  };
+
   empty = {
     plugins.treesitter.enable = true;
   };
 
-  nonix = {
-    # TODO: See if we can build parsers (legacy way)
-    tests.dontRun = true;
+  emptyGrammarPackages = {
```

```suggestion
  empty-grammar-packages = {
```

We normally kebab-case our test-case names